diff --git test/util/test_modelzoo.py test/util/test_modelzoo.py
index 69cc56f..13a35ff 100644
--- test/util/test_modelzoo.py
+++ test/util/test_modelzoo.py
@@ -89,11 +89,11 @@ class TestModelzoo(unittest.TestCase):
 
         return output_path
 
-    @unittest.expectedFailure
+    # @unittest.expectedFailure
     def test_export_single_channel(self):
         self._test_export(1, "single-chan")
 
-    @unittest.expectedFailure
+    # @unittest.expectedFailure
     def test_export_multi_channel(self):
         self._test_export(4, "multi-chan")
 
